donald_wade/week_02/styles2.css

```diff
@@ -0,0 +1,61 @@
+* {
+  /*border: 1px dotted red;*/
+  box-sizing: border-box;
+}
+body {
+  height: 100vh;
+  width: 100vw;
+}
+
+.box {
+  font-size: 3em;
+  padding: 5%;
+  width: 50vw;
+  height: 50vh;
+  transition: all 0.8s ease;
+}
```

Comment:
try to keep to one nomination when refer to sizes (paddings, margins, width, height, font-size etc), if you are using em use them for all styling, if using % use it etc. Best recommendation is to use em or rem.